pkg/hub/options/constant.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2021 The Clusternet Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"time"
+)
+
+// default values
+const (
+	openAPITitle = "Clusternet"
+
+	DefaultClusterMonitorPeriod      = 20 * time.Second
```

Comment:
Better not set such period settings on hub side,
1. Hub will manage multiple child clusters. It is not feasible to have such overall settings. And also it is not feasible to customize these settings and restart hub time by time.
2. And agents running in child clusters will report heartbeats in a configurable period settings. Thus settings on hub side CAN NOT accurately monitor every cluster.

Reply:
@huxiaoliang WDYT

Reply:
@dixudx, nice catch, I am agree with you to make the setting on agent side

Reply:
ok, for newest implement, hub will use HeartbeatFrequencySeconds to calculate cluster unknown conditions. thanks for the suggestion.